core/src/test/scala/unit/kafka/common/TopicTest.scala

```diff
@@ -17,6 +17,7 @@
 
 package kafka.common
 
+import org.apache.kafka.common.errors
```

Comment:
Nit: Same comment as earlier about using the package as the import instead of the single class.